Update to remove OSG CA stuff (SW-3143)

sthapa opened the pull request.

brianhlin commented on docs/security/user-certs.md:
```diff
@@ -10,13 +10,23 @@ Getting a User Certificate
-Currently you can get a certificate from either the OSG CA or from CILogon.
-After May 31st, you will only be able to get a certificate issued by CILogon.
-Other CAs may be used; if your virtual organization (VO) requires that you get a certificate from a different CA, [contact your VO Support Center](http://www.opensciencegrid.org/?pid=1000187) for instructions.
+You need an user certificate if you are going to interact directly with OSG resources or infrastructure. 
```
a user*

sthapa replied:
fixed

brianhlin commented on docs/security/user-certs.md:
```diff
@@ -10,13 +10,23 @@ Getting a User Certificate
+You need an user certificate if you are going to interact directly with OSG resources or infrastructure. 
+Examples of this would be tasks like managing OASIS, directly running jobs on OSG resources, interactingly directly with 
+OSG storage elements, or updating or querying OSG site information (such as site topology).
+Currently, you can get am user certificate from CILogon.
```
am -> an

sthapa replied:
fixed

brianhlin commented on docs/security/user-certs.md:
```diff
@@ -10,13 +10,23 @@ Getting a User Certificate
-Other CAs may be used; if your virtual organization (VO) requires that you get a certificate from a different CA, [contact your VO Support Center](http://www.opensciencegrid.org/?pid=1000187) for instructions.
+You need an user certificate if you are going to interact directly with OSG resources or infrastructure. 
+Examples of this would be tasks like managing OASIS, directly running jobs on OSG resources, interactingly directly with 
+OSG storage elements, or updating or querying OSG site information (such as site topology).
```
You don't need a cert to update/query OSG site info. Only to access private contact information (e.g. email addresses/phone numbers/DNs)

sthapa replied:
updated

brianhlin commented on docs/security/user-certs.md:
```diff
@@ -10,13 +10,23 @@ Getting a User Certificate
 ### Know your responsibilities
 
-When you request a certificate your provide some public personal information about yourself; *name, email address, phone number*, and which *VO* (virtual organization) you belong to. If any of this information changes after you have your certificate you should notify OSG (and probably VO) of the changes. For the OSG RA send email to osg-ra at opensciencegrid dot org.
+When you receive an user certificate, 
```
Delete this line

sthapa replied:
done

brianhlin commented on docs/security/user-certs.md:
```diff
@@ -10,13 +10,23 @@ Getting a User Certificate
+number*, and which *VO* (virtual organization) you belong to. 
+If any of this information changes after you have your certificate you should notify CILogon (and probably VO) of the changes. 
+In addition, if your account or user certificate is compromised, you **must** notify the issuer of your certificate. 
+For the CILogon RA send email to ca at ciloogon dot org. 
```
This can just be `<mailto:ca@cilogon.org>`

sthapa replied:
updated

brianhlin commented on docs/security/user-certs.md:
```diff
@@ -10,13 +10,23 @@ Getting a User Certificate
+When you receive an user certificate, 
+When you request a certificate, you provide some public personal information about yourself; *name, email address, phone
+number*, and which *VO* (virtual organization) you belong to. 
+If any of this information changes after you have your certificate you should notify CILogon (and probably VO) of the changes. 
```
You don't need to provide any of the above in the new cilogon world order. The subsequent sentence can also be removed

sthapa replied:
done

brianhlin commented on docs/security/user-certs.md:
```diff
@@ -25,36 +35,23 @@ Your **name** and **email** address are encoded into the signed certificate and
 Follow the steps below to get an user certificate:
 
-1. First, either search for your institution and select it or scroll through list and do the same. <br>![Institution Selection](/img/cilogon_select_idp.png)
+1. First, either search for your institution and select it or scroll through list and do the same. If your institution is not on the list, please contact your institution's IT support to see if they can support CILogon. <br>![Institution Selection](/img/cilogon_select_idp.png).  
```
There's some extra trailing space at the nd of the sentence. Could you also put the sentence on a newline?

sthapa replied:
done

brianhlin commented on docs/security/user-certs.md:
```diff
@@ -25,36 +35,23 @@ Your **name** and **email** address are encoded into the signed certificate and
 ### PKCS12 (.p12) vs PEM format
 
-Your certificate and key pair can be stored as separate files, by default named `~/.globus/usercert.pem` and `~/.globus/userkey.pem`, or they can be bundled in a single file in PKCS12 (Public Key Cryptography Standard \#12), by default named `~/.globus/usercred.p12`. All OSG user tools works with both formats. Unless you specify a name in the command, they look first for the certificate/key pair  ...
+Your certificate and key pair can be stored as separate files, by default named `~/.globus/usercert.pem` and `~/.globus/userkey.pem`, or they can be bundled in a single file in PKCS12 (Public Key Cryptography Standard \#12), by default named `~/.globus/usercred.p12`. All OSG user tools works with both formats. Unless you specify a name in the command, they look first for the certificate/key pair  ...
```
- Let's change PKCS12 into a link to the wikipedia page https://en.wikipedia.org/wiki/PKCS_12. Then get rid of the parenthetical.
- I don't think it's true that all OSG user tools work with both formats. 
- The last sentence is repetitive

sthapa replied:
Done

sthapa commented on docs/security/user-certs.md:
```diff
@@ -10,13 +10,23 @@ Getting a User Certificate
-After May 31st, you will only be able to get a certificate issued by CILogon.
-Other CAs may be used; if your virtual organization (VO) requires that you get a certificate from a different CA, [contact your VO Support Center](http://www.opensciencegrid.org/?pid=1000187) for instructions.
+You need an user certificate if you are going to interact directly with OSG resources or infrastructure. 
+Examples of this would be tasks like managing OASIS, directly running jobs on OSG resources, interactingly directly with 
```
fixed

brianhlin commented:
@sthapa I think you missed this comment: 

> In addition to the comments, the PKCS12 section just needs to be redone entirely. Replace it with a section on what PKCS12 is and step-by-step instructions on how to extract the cert and key.

sthapa commented:
Updated the PKCS12 section, dropped the part about troubleshooting the PKCS12 ca chain expiration but I can add a rewritten version of that in if needed.

sthapa commented:
@brianhlin  mind looking at my updates?

brianhlin commented on docs/security/user-certs.md:
```diff
@@ -44,74 +44,34 @@ Follow the steps below to get an user certificate:
+### Certificate formats
+
+Your user certficate can be stored in a few different formats.
+The two most common formats used on OSG are the [PKCS12](https://en.wikipedia.org/wiki/PKCS_12) and
```
"on the OSG" -> "in the OSG"

sthapa replied:
fixed

brianhlin commented on docs/security/user-certs.md:
```diff
@@ -44,74 +44,34 @@ Follow the steps below to get an user certificate:
+
+To convert a PKCS12 file to  PEM files, do the following.  
+   
+1. First, extract your user certificate from your PKCS12 file.  We assume that the PKCS12 file is called `usercred.p12` and output the PEM certificate to `usercert.pem`.
```
Let's break up the sentences to separate lines and also add something about entering the password specified when the cert was requested

sthapa replied:
done

brianhlin commented on docs/security/user-certs.md:
```diff
@@ -44,74 +44,34 @@ Follow the steps below to get an user certificate:
+        Enter Import Password:
+        MAC verified OK
+   
+1. Second, extract the private key. The private key will be saved to userkey.pem
```
Let's mention and differentiate between the two different password prompts

sthapa replied:
done

brianhlin commented:
LGTM!
